### ISSUE TITLE
Please update websockets to 2.0.0.

### ISSUE THREAD

**Issue by Romick2005** [HenningM/express-ws#54]
Can you please try to switch ws to 2.0.0-beta.1?

**Comment by aichholzer** [comment]
**Please update.**
There is something weird going on here with the current version of `ws` being used. It keeps throwing this error on incoming messages:

```
/node_modules/express-ws/node_modules/ws/lib/Receiver.js:306
if (mask != null && buf != null) bufferUtil.unmask(buf, mask);
TypeError: Cannot read property 'unmask' of undefined
```
the `Receiver.js` is doing this (line 11):
```
bufferUtil = require('./BufferUtil').BufferUtil
```
and `BufferUtil.js` does this:
```
try {
  module.exports = require('bufferutil');
} catch (e) {
  module.exports = require('./BufferUtil.fallback');
}
```
which results in the error since `bufferutil.BufferUtil` (as used in `Receiver.js`)  does not exist.

This works as expected and the code is also doing the right thing in the latest `ws` version.

**Comment by HenningM** [comment]
Should be fixed in [version 3.0.0](https://github.com/HenningM/express-ws/releases/tag/v3.0.0).
